changelog/1444.feature.rst

```diff
@@ -0,0 +1,5 @@
+Migrated
+`~plasmapy.formulary.dimensionless.Debye_number`, and
+`~plasmapy.formulary.dimensionless.Hall_parameters` from
```

Comment:
```suggestion
`~plasmapy.formulary.dimensionless.Hall_parameter` from
```

Reply:
I knew I'd screw up one of these! My fingers don't like typing parametes (<- see I even missed the "r" on that). 😭